miso-web/src/main/webapp/scripts/tile.js

```diff
@@ -21,116 +21,67 @@
  * *********************************************************************
  */
 
-TileTarget = {};
-
-Tile = (function() {
-  var showPane = function(elementId, target, inner) {
-    var div = document.getElementById(elementId);
-    div.setAttribute("class", "dashboard_widget");
-    while (div.hasChildNodes()) {
-      div.removeChild(div.lastChild);
-    }
+Tile = {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add the "let", "const" or "var" keyword to this declaration of "Tile" to make it explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS2703)